src/types/editor.ts

```diff
@@ -89,6 +89,8 @@ type DisabledField =
 	| 'equipment'
 	| 'meetingRoom';
 
+export type CalendarOrganizer = any;
```

Comment:
'unknown' would probably be a better fit

Reply:
I agree but it will cause errors in the entire module which are really difficult to tackle right now. That is why I decided to export that type, once I will be able to set the proper type, it will be used throughout the entire app. Thanks for the suggestion, very much appreciated!